pkg/preflight/assets/preflights.yaml

```diff
@@ -6,6 +6,12 @@ metadata:
 spec:
   collectors:
     - clusterResources: {}
+    - run:
```

Comment:
the in-cluster 'run' collector is deprecated, replaced by runPod.

I think you're looking for the host collector though - https://troubleshoot.sh/docs/host-collect-analyze/run/

Reply:
Done thank you.